client/admin/components/users/index.vue

```diff
@@ -34,6 +34,7 @@
             <td>
               <v-icon @click="showUserDialog(item)" small>mdi-pencil</v-icon>
               <v-icon @click="removeUser(item)" small class="ml-3">mdi-delete</v-icon>
+              <v-icon @click="reinviteUser(item)" small class="ml-3">mdi-email</v-icon>
```

Comment:
Remove reinvite from user list and make it available solely from user modal (top right corner).